### PR TITLE
Splitwiseclone html

### PR DESCRIPTION
**Pull request by saba024**
Add html and css.

### CODE REVIEW

**Comment by saba024** [comment]
https://saba024.github.io/Front-end_labs/index.html

File: index.html [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+                <a href="./index.html" class="header__logo">
+                    SplitwiseClone
+                </a>
+                <div class="lang-menu">
+					<div class="selected-lang">
+						English
+					</div>
+					<ul>
+						<li>
+							<a class="ru">Russian</a>
```
Comment (gmltA) [inline index.html]: How's that a link?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" type="text/css" href="./style.css">
+	<meta charset="UTF-8">
+	<title>Document</title>
+</head>
+<body>
+	<header class="header">
```
Comment (gmltA) [inline index.html:9]: image

Everything went wrong!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+                        <div class="budget-expenses-percentage">45%</div>
+                    </div>
+            </div>
+		</div>
+	</div>
+
+	<div class="content">
+		<div class="tabs">
+			<nav class="tabs__items">
+				<a href="#tab_01" class="tabs_item"><span class="tab_name">Expenses</span></a>
```
Comment (gmltA) [inline index.html]: This is also not a link.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+						<div class="view_item">
+							<div class="vi_left" >
+								<img src="./food.jpg" alt="food">
+							</div>
+							<div class="vi_right">
+								<p class="title">Burger</p>
+								<div class="expense_amount">50$</div>
+								<img src="./receipt.jpg" alt="receipt">
+								<p class="content">Bob -----> Greg, Bill</p>
+								<div class="btn_edit">Edit</div>
```
Comment (gmltA) [inline index.html]: These should be `button`s.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+								<p class="content">Greg -----> Bob, Bill</p>
+								<div class="btn_edit">Edit</div>
+								<div class="btn_delete">Delete</div>
+							</div>
+						</div>
+					</div>
+				</div>
+
+				<div id="tab_02" class="tabs__block">
+					<div class="view_wrap expense-list">
```
Comment (gmltA) [inline index.html]: If it is a list, why do you use `div`?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+			<h2 class="title">Add new Expense</h2>
+			<label for="name" class="name">Who paid</label>
+			<div class="name-select" >
+				<select>
+					<option value="0">Greg</option>
+					<option value="1">Bob</option>
+					<option value="2">Bill</option>
+				</select>
+			</div>
+			<label for="amount" class="amount">Amount</label>
```
Comment (gmltA) [inline index.html]: Labels for what?

**Review by gmltA — CHANGES_REQUESTED** [review]

File: index.html [hunk 7]
```diff
@@ -62,15 +50,15 @@
...
+				<button class="tabs_item" onclick="openOption(event, 'tab_01')"><span class="tab_name">Expenses</span></button>
+				<button class="tabs_item" onclick="openOption(event, 'tab_02')"><span class="tab_name2">Debts</span></a>
+				<button class="tabs_item" onclick="openOption(event, 'tab_03')"><span class="tab_name3">Members</span></a>
+				<button class="tabs_item" onclick="openOption(event, 'tab_04')"><span class="tab_name4">Activity</span></a>
 			</nav>
 			<div class="tabs__body">
 				<div id="tab_01" class="tabs__block">
 					<p><button class="add_expense">Add Expenses</button></p>
-					<div class="view_wrap expense-list">
+					<ul class="view_wrap expense-list">
```
Comment (gmltA) [inline index.html:59]: And where are your list items then?

File: index.html [hunk 8]
```diff
@@ -177,21 +165,21 @@ <h2 class="title">Add new Expense</h2>
 					<option value="2">Bill</option>
 				</select>
 			</div>
-			<label for="amount" class="amount">Amount</label>
+			<label class="amount">Amount</label>
```
Comment (gmltA) [inline index.html]: U kidding me? It is also wrong. Labels are meant for inputs!

**Review by gmltA — CHANGES_REQUESTED** [review]

**Comment by gmltA** [comment]
image

This is still broken. Plus you are not allowed to use `float`.

File: index.html [hunk 9]
```diff
@@ -165,21 +163,21 @@ <h2 class="title">Add new Expense</h2>
...
-			<label class="concept">Concept</label>
-			<input type="text" name="concept" class="concept_input" placeholder="Expense name">
-			<label class="category">Category</label>
-			<div class="category-select" >
+			<label class="amount" for="amount">Amount</label>
+			<input type="value" name="amount" class="amount_input" id="amount" placeholder="0$">
+			<label for="concept" class="concept">Concept</label>
+			<input type="text" name="concept" class="concept_input" id="concept" placeholder="Expense name">
+			<label for="category-select" class="category">Category</label>
+			<div class="category-select" id="category-select" >
```
Comment (gmltA) [inline index.html]: The heck is this?

File: index.html [hunk 10]
```diff
@@ -165,21 +163,21 @@ <h2 class="title">Add new Expense</h2>
...
 					<option value="0">Food</option>
 					<option value="1">Home</option>
 					<option value="2">Tickets</option>
 				</select>
 			</div>
-			<label class="shared">Shared</label>
-			<button class="share_equally">share equally</button>
-			<label class="add_receipt">Add Receipt</label>
+			<label for="shared" class="shared">Shared</label>
+			<button class="share_equally" id="shared">share equally</button>
```
Comment (gmltA) [inline index.html]: And this.

**Review by gmltA — CHANGES_REQUESTED** [review]

**Review by gmltA — APPROVED** [review]
Okay. That's a <kbd>6</kbd>.

**Pull request merged**